.travis.yml

```diff
@@ -26,16 +23,33 @@ before_script:
 
 install:
   - sudo apt-get remove ipython
-  - pip install --upgrade pip
-  - pip install --upgrade setuptools
-  - pip install --upgrade wheel
+  - pip install --upgrade pip setuptools wheel
   - pip install $IPYTHON
   - pip install --only-binary=numpy,scipy numpy==$NUMPY_VERSION scipy
   - pip install -e '.[MongoTrials, SparkTrials, ATPE, dev]'
   - pip install nose
```

Comment:
I think we can lose nose if we are running pytest.
```suggestion
```

Reply:
there are some tests in each style, so nose is needed as it's imported